ynr/apps/elections/views.py

```diff
@@ -68,9 +70,25 @@ class ElectionListView(TemplateView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["elections_and_posts"] = Election.group_and_order_elections(
-            include_postextraelections=True, include_noncurrent=False
+
+        qs = (
+            PostExtraElection.objects.filter(election__current=True)
+            .select_related("election", "post")
+            .prefetch_related("suggestedpostlock_set")
+            .prefetch_related("officialdocument_set")
+            .annotate(memberships_count=Count("membership"))
+            .order_by("election__election_date", "election__name")
         )
+
+        f = BallotFilter(self.request.GET, qs)
+
+        context["filter"] = f
+        context["queryset"] = f.qs
+        if f.data.get("review_required") == "suggestion":
+            context["queryset"] = context["queryset"][:10]
```

Comment:
Nope - I think I'm just going to fast and getting confused/looking at the wrong bit of code I think..

But then my follow-on question is: do we need/want this? In the `/elections` view, presumably we just want to always show all elections?

Reply:
Oh yeah, need to remove that 👍